main/tests/test_render.cpp

```diff
@@ -182,38 +182,38 @@ class TestMainLoop : public MainLoop {
 		RID lightaux;
 
 		lightaux = vs->directional_light_create();
-		//vs->light_set_color( lightaux, VisualServer::LIGHT_COLOR_AMBIENT, Color(0.0,0.0,0.0) );
+
 		vs->light_set_color(lightaux, Color(1.0, 1.0, 1.0));
-		//vs->light_set_shadow( lightaux, true );
+
 		light = vs->instance_create2(lightaux, scenario);
 		Transform lla;
-		//lla.set_look_at(Vector3(),Vector3(1,-1,1),Vector3(0,1,0));
+
 		lla.set_look_at(Vector3(), Vector3(-0.000000, -0.836026, -0.548690), Vector3(0, 1, 0));
 
 		vs->instance_set_transform(light, lla);
 
 		lightaux = vs->omni_light_create();
-		//vs->light_set_color( lightaux, VisualServer::LIGHT_COLOR_AMBIENT, Color(0.0,0.0,1.0) );
+
 		vs->light_set_color(lightaux, Color(1.0, 1.0, 0.0));
 		vs->light_set_param(lightaux, VisualServer::LIGHT_PARAM_RANGE, 4);
 		vs->light_set_param(lightaux, VisualServer::LIGHT_PARAM_ENERGY, 8);
-		//vs->light_set_shadow( lightaux, true );
-		//light = vs->instance_create( lightaux );
+
+
 
 		ofs = 0;
 		quit = false;
 	}
 	virtual bool iteration(float p_time) {
 
 		VisualServer *vs = VisualServer::get_singleton();
-		//Transform t;
-		//t.rotate(Vector3(0, 1, 0), ofs);
-		//t.translate(Vector3(0,0,20 ));
-		//vs->camera_set_transform(camera, t);
+
+
+
+
```

Comment:
Why do some of these replace code with empty lines? There's 4 in a row here

Reply:
It seems like I didn't run clang-format correctly on all files.